tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -1032,8 +952,9 @@ def _stage_create_topics_adjusted(self, profile, producer_node_only=False):
             produce_nodes = nodes_available
             consume_nodes = 0
             # get topic count
-            produce_node_topic_count = profile.topic_count // produce_nodes
-            consume_node_topic_count = 0
+            produce_node_client_count = profile.topic_count // (
```

Comment:
Should probably round up, not down or at least do some basic sanity checks here that the integer division isn't too bad, e.g., we don't want 1000 topic_count, 100 topics / client and 6 nodes to give 1 client after rounding, i.e., only 600 topics are created.

Reply:
Just to make it simple I'll just switch to regular division and assert that the result is an integer. I configured the tests to always have these values be integers anyhow.